helpers/mongoDB-utils.js

```diff
@@ -83,10 +86,33 @@ export const fetchCategories = async () => {
   try {
     const client = await DBConnection();
     const fetchedCategories = await getAllCategories(client);
-    console.log(fetchedCategories);
-    client.close();
```

Comment:
Please include client.close, you removed it by mistake. @Mpho-vincent-makgwe

Reply:
Added it back...everything it's fine now.

Reply:
please check for more that wasnt the only one, think there 1 code line above this one too.

Reply:
Used finally to close the client.
Thank you